internal/runtime/runtime_test.go

```diff
@@ -76,7 +76,8 @@ func TestProcessMessageBusRequest(t *testing.T) {
 
 	runtime := NewGolangRuntime("", nil, dic)
 	runtime.SetDefaultFunctionsPipeline([]interfaces.AppFunction{dummyTransform})
-	result := runtime.ProcessMessage(context, envelope, runtime.GetDefaultPipeline())
+
+	_, result, _ := runtime.DecodeMessage(context, envelope)
 	require.NotNil(t, result)
 	assert.Equal(t, expected, result.ErrorCode)
```

Comment:
would it be beneficial to add an `assert.Err(...)` for the err here? And for all subsequent calls to `runtime.DecodeMessage(...)`?